schema.yaml

```diff
@@ -89,28 +89,28 @@ schema;game:
 schema;genres:
   type: str
   enum: [
+    'Action',
+    'Adventure',
+    'Arcade',
+    'Artillery',
+    'Educational',
+    'Fighting',
     'FPS',
     'MMORPG',
     'MOBA',
+    'Platform',
+    'Puzzle',
+    'Racing',
     'RPG',
     'RTS',
+    'Sports',
     'TBS',
     'TPS',
-    'action',
-    'adventure',
-    'arcade',
-    'artillery',
-    'educational',
-    'fighting',
-    'platform',
-    'puzzle',
-    'racing',
-    'rhythm',
-    'roguelike',
-    'shmup',
-    'simulation',
-    'sports',
-    'horror',
+    'Rhythm',
+    'Roguelike',
+    'Shmup',
+    'Simulation',
+    'Strategy',
   ]
 
 schema;subgenres:
```

Comment:
Me and @wonea are against the use of subgenres https://github.com/piranha/osgameclones/pull/411#issuecomment-270892869

I would remove all subgenre information and even themes - they pollute the list without any added benefit as far as I'm concerned. I don't think people come here expecting to search for "crime" games or even "fantasy". We don't see theme and sub-genre information on Wikipedia-listed games. If a subgenre or theme is important enough to exist is should be added as a main genre. 

The list is already big enough as it is without all that extra clutter of dubious value to visitors. If someone can make a strong case of why it is of benefit to the project while at the extra cost of inputting them and added complexity overall, I'll be happy to be persuaded.